examples/01-basic/11-removing-block-option/App.tsx

```diff
@@ -0,0 +1,54 @@
+import "@blocknote/core/fonts/inter.css";
+import { useCreateBlockNote } from "@blocknote/react";
+import { BlockNoteView } from "@blocknote/mantine";
+import "@blocknote/mantine/style.css";
+import { BlockNoteSchema, defaultBlockSpecs, defaultInlineContentSpecs,  defaultStyleSpecs } from "@blocknote/core";
+
+export default function App() {
+  //create custom schema
+  let editedSchema : any =  {}
+
+  //define the blocks which are not allowed
+  const unwantedBlocks = ['audio', 'image']
+
+  //Loop thru default schema and remove the unwanted blocks
+  Object.entries(defaultBlockSpecs).forEach(([blockName, blockSchema])=>{
+    if(!unwantedBlocks.includes(blockName)){
+
+      //only allow to to pass if the blockName is not unwanted
+      editedSchema[blockName] = blockSchema
+    }
+  })
+
+  //now create the schema
+  const schema = BlockNoteSchema.create({
+    blockSpecs: {
```

Comment:
does the following actually work?
```
blockSpecs: {
...defaultBlockSpecs,
audio: undefined
}```

or alternatively:

```
{ audio, ...keepBlockSpecs } = defaultBlockSpecs

blockSpecs: keepBlockSpecs;
```

I'd prefer the first option, but both are a bit cleaner / simpler then manually looping the specs imo